source/agora/common/BitMask.d

```diff
@@ -62,27 +62,32 @@ public struct BitMask
     }
 
     /// Bytes to hold bits to indicate if a validator has signed
-    private ubyte[] bytes;
+    private ubyte[] _bytes;
+
+    /// readonly public access to bytes
+    public const(ubyte[]) bytes() const scope @nogc
+    {
+        return this._bytes;
+    }
```

Comment:
Why does this need to be exposed ?

Reply:
I thought I needed it to access `this._bytes` in the `copyFrom` parameter `rhs`. But checking now it is not required. I'll update.